### ISSUE TITLE
[User Story] User trying WebAssembly "Hello, world" example

### ISSUE THREAD

**Issue by yurydelendik** [wasdk/WebAssemblyStudio#32]
As a person who does not know WebAssembly,
I want the ability to quickly see a simple application, 
So that I may run it and also make a simple modification to experiment with the example.

**Comment by haxxorsid** [comment]
Hmm

**Comment by thuselem** [comment]
I would like to write this example. Should it be included with the templates? I see `hello_world_c` in there.

**Comment by yurydelendik** [comment]
> Should it be included with the templates? 

Yes, it should. I think users will appreciate the small hello world wat application/example.
